portfolio/src/main/webapp/script.js

```diff
@@ -49,5 +49,10 @@ function createListElement(text) {
   return liElement;
 }
 
+function deleteComments(){
+  const params = new URLSearchParams();
+  fetch('/delete-data', {method: 'POST', body: params});
+}
+
```

Comment:
i could be wrong, but does this clear the comments from the page after the comments are deleted??

Reply:
No, this function tells the servlet to delete the comments.

Reply:
so, if there are comments on the page and then you press delete, the comments will no longer be on the page?

Reply:
In my case, they stay until you change the number of comments you want to see, then they disappear and if you comment again, they go away.

Reply:
oh! i gotchu.